docs/installation.rst

```diff
@@ -17,7 +17,7 @@ After `installing Anaconda <https://docs.anaconda.com/anaconda/install/index.htm
 
     $ conda install -c conda-forge tiatoolbox
 
-Please note that conda-forge installation support is limited on Windows as openslide binaries are not supported on official conda channels. An alternate way to install using conda on Windows could be to install it in `WSL2 with CUDA support <https://docs.microsoft.com/en-us/windows/ai/directml/gpu-cuda-in-wsl>`_.
+Please note that conda-forge installation support is limited on Windows as openslide binaries are not supported on official conda channels. An alternate way to install using conda on Windows could be to install it in `WSL2 with CUDA support <https://docs.microsoft.com/en-us/windows/ai/directml/gpu-cuda-in-wsl>`_. We have found WSL2 to be faster to run Python codes in some cases and therefore we would recommend this option.
```

Comment:
We have found WSL2 to be faster to run Python codes in some cases and therefore we would recommend this option.

This is OK, but I prefer:
In some cases, WSL2 runs faster on Python codes iand therefore we recommend this option.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/454/commits/0fe39b442fe592ee78767aec2c26e62dc8cac51b